pkg/cloud/cloud.go

```diff
@@ -2,25 +2,29 @@ package cloud
 
 import (
 	"context"
+	"fmt"
+	"strings"
 	"time"
 
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
-	InstanceTag = "multi-platform-instance"
-	OKState     = VMState("OK")
-	FailedState = VMState("FAILED")
+	InstanceTag   = "multi-platform-instance"
+	TaskRunTagKey = "taskRunID"
+	OKState       = VMState("OK")
+	FailedState   = VMState("FAILED")
```

Comment:
```suggestion
	OKState       VMState = "OK"
	FailedState  VMState = "FAILED"
```
If you prefer, this format should also work